assets/localization-form.js

```diff
@@ -6,28 +6,91 @@ if (!customElements.get('localization-form')) {
         super();
         this.elements = {
           input: this.querySelector('input[name="locale_code"], input[name="country_code"]'),
-          button: this.querySelector('button'),
+          button: this.querySelector('button.localization-form__select'),
           panel: this.querySelector('.disclosure__list-wrapper'),
+          search: this.querySelector('input[name="country_filter"]'),
+          closeButton: this.querySelector('.country-selector__close-button'),
+          resetButton: this.querySelector('.country-filter__reset-button'),
+          searchIcon: this.querySelector('.country-filter__search-icon'),
+          liveRegion: this.querySelector('#sr-country-search-results'),
         };
-        this.elements.button.addEventListener('click', this.openSelector.bind(this));
-        this.elements.button.addEventListener('focusout', this.closeSelector.bind(this));
         this.addEventListener('keyup', this.onContainerKeyUp.bind(this));
+        this.addEventListener('keydown', this.onContainerKeyDown.bind(this));
+        this.addEventListener('focusout', this.closeSelector.bind(this));
+        this.elements.button.addEventListener('click', this.openSelector.bind(this));
+
+        if (this.elements.search) {
+          this.elements.search.addEventListener('keyup', this.filterCountries.bind(this));
+          this.elements.search.addEventListener('focus', this.onSearchFocus.bind(this));
+          this.elements.search.addEventListener('blur', this.onSearchBlur.bind(this));
+          this.elements.search.addEventListener('keydown', this.onSearchKeyDown.bind(this));
+        }
+        if (this.elements.closeButton) {
+          this.elements.closeButton.addEventListener('click', this.hidePanel.bind(this));
+        }
+        if (this.elements.resetButton) {
+          this.elements.resetButton.addEventListener('click', this.resetFilter.bind(this));
+          this.elements.resetButton.addEventListener('mousedown', (event) => event.preventDefault());
+        }
 
         this.querySelectorAll('a').forEach((item) => item.addEventListener('click', this.onItemClick.bind(this)));
       }
 
       hidePanel() {
         this.elements.button.setAttribute('aria-expanded', 'false');
         this.elements.panel.setAttribute('hidden', true);
+        if (this.elements.search) {
+          this.elements.search.value = '';
+          this.filterCountries();
+          this.elements.search.setAttribute('aria-activedescendant', '');
+        }
+        document.body.classList.remove('overflow-hidden-mobile');
```

Comment:
You will probably need to update this one as well to make sure it's added when needed and not on tablet sizes.

Reply:
This line removes the class so I think it's fine as is